go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -735,6 +735,19 @@ var DoltScripts = []queries.ScriptTest{
 			},
 		},
 	},
+	{
```

Comment:
Maybe a bit of a battery of float tests as well? I'm concerned about our coverage since this was only caught in fuzzer testing.